### DISCUSSION TITLE
API ../forecast urls not returning precipitation/qpf and others

### DISCUSSION THREAD

**Discussion by nicupavel** [weather-gov/api#207] [Q&A]
Hi,
Looking at the data returned by forecast or forecast/hourly (either with /gridpoints/ or /points) there are no entries for precipitation amount (qpf) and others like pressure, rh etc.
Is there a way to obtain precipitation forecast and others ?
If not is this planned for the future ?

**Comment by JJKraw** [comment]
Have you looked at the qualitativePrecipitation field returned under URLs of the form:

gridpoints/{office ID}/{gridpoints}/  ?

**Comment by nicupavel** [comment]
Thanks for the reply. There doesn't seem to be this kind of API path (it gives an error response) the only ones with /gridpoints are:

```
GET /gridpoints/{wfo}/{x},{y}
GET /gridpoints/{wfo}/{x},{y}/forecast
GET /gridpoints/{wfo}/{x},{y}/forecast/hourly
GET /gridpoints/{wfo}/{x},{y}/stations
```
None of these have qualitativePrecipitation or quantitativePrecipitation

**Comment by JJKraw** [comment]
Meant to include the points (your first example).

If I pull:

  https://api.weather.gov/gridpoints/BOX/64,79

I get values under quantitativePrecipitation. If the values do not appear for other locations, it's probably because the WFO didn't provide the data (I believe this is the raw data from the WFO).

**Comment by nicupavel** [comment]
Yes thank you, that url gives all the information needed which is raw forecast data.
